pkg/v3/resource/chartconfig/desired.go

```diff
@@ -14,20 +19,53 @@ const (
 
 // GetDesiredState returns all desired ChartConfigs for managed guest resources.
 func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interface{}, error) {
-	// TODO: Add ChartConfigs as we migrate components from k8scloudconfig.
-	/*
-		clusterGuestConfig, err := r.toClusterGuestConfigFunc(obj)
-		if err != nil {
-			return nil, microerror.Mask(err)
-		}
-
-		clusterConfig, err := prepareClusterConfig(r.baseClusterConfig, clusterGuestConfig)
-		if err != nil {
-			return nil, microerror.Mask(err)
-		}
-	*/
+	clusterGuestConfig, err := r.toClusterGuestConfigFunc(obj)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	clusterConfig, err := prepareClusterConfig(r.baseClusterConfig, clusterGuestConfig)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
 
 	desiredChartConfigs := make([]*v1alpha1.ChartConfig, 0)
+	{
+		chartConfig := newKubeStateMetricsChartConfig(clusterConfig, r.projectName)
+		desiredChartConfigs = append(desiredChartConfigs, chartConfig)
+	}
 
 	return desiredChartConfigs, nil
 }
+
+func newKubeStateMetricsChartConfig(clusterConfig cluster.Config, projectName string) *v1alpha1.ChartConfig {
+	chartName := "kubernetes-kube-state-metrics-chart"
+	channelName := "0-1-stable"
+	releaseName := "kube-state-metrics"
```

Comment:
This is the chart that will be installed by chart-operator.

Reply:
So here it goes... Exciting <3